tox.ini

```diff
@@ -2,7 +2,7 @@
 # fakeroot -u tox --recreate
 
 [tox]
-envlist = py{35,36,37,38},flake8
+envlist = py{35,36,37,38},windows,flake8
```

Comment:
if one wants to run all the tests (on linux), one could just invoke `tox` (without arguments).

with this change, this would also run the `windows` tests, which does not make sense.

Reply:
and it would be similarly broken on windows, too.

Reply:
if the point is just avoiding fuse failing to install on platforms not supporting fuse, the hack i did for some BSDs was to overwrite `requirements.d/fuse.txt` with an empty file.

Reply:
Where can I find this hack? I search everywhere but I could not find it.
I don't like to hide this somewhere in some scripts and would prefere a clean solution.

Reply:
What do you thing about the solution used below? Maybe this can be improved further, but at least in this way it is clear that on windows something different happens.

Reply:
This is no longer relevant as the testing will be done later.